### PR TITLE
Small refactoring before implementing adding CoinJoin account UI

### PR DESCRIPTION
**Pull request by matejkriz**
## Description

Refactoring only, no logic changes.

## Related Issue

Preparation for https://github.com/trezor/trezor-suite/issues/5873

### CODE REVIEW

File: packages/suite/src/components/wallet/AccountsMenu/AccountItemSkeleton.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { SkeletonCircle, SkeletonRectangle, Stack } from '@suite-components/Skeleton';
+import { useLoadingSkeleton } from '@suite-hooks';
+import { AccountHeader, Left, Right } from './AccountItem';
+
+export const AccountItemSkeleton = (props: { animate?: boolean }) => {
```
Comment (dahaca) [inline packages/suite/src/components/wallet/AccountsMenu/AccountItemSkeleton.tsx]:
```suggestion
export const AccountItemSkeleton = ({ animate }: AccountItemSkeletonProps) => {
```

**Review by dahaca — APPROVED** [review]

File: packages/suite/src/components/wallet/AccountsMenu/AccountItemSkeleton.tsx (hunk 1 above)
Comment (matejkriz) [reply packages/suite/src/components/wallet/AccountsMenu/AccountItemSkeleton.tsx]: it was just a draft so far, please check the second commit

File: packages/suite/src/components/suite/modals/AddAccount/components/AddButton.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import { Button, Tooltip } from '@trezor/components';
+import { Translation } from '@suite-components';
+
+interface ButtonProps {
+    disabledMessage: React.ReactNode;
+    handleClick: () => void;
+}
+
+export const AddButton = ({ disabledMessage, handleClick }: ButtonProps) => (
+    <Tooltip maxWidth={285} content={disabledMessage}>
```
Comment (dahaca) [inline packages/suite/src/components/suite/modals/AddAccount/components/AddButton.tsx:11]: I'd add a question icon tooltip when it's disabled or underline text at least, it's not clear there is an interaction (tooltip)

Comment (matejkriz) [reply packages/suite/src/components/suite/modals/AddAccount/components/AddButton.tsx:11]: yeah, would be nice to have some global solution for "disabled" buttons – I would imagine not to disable the buttons at all and display meaningful validation message when user click on button and some conditions to do the action are not met..

I don't think it makes sense to change it on on place in one way right now.

**Pull request merged**